apps/style/javalab/style.scss

```diff
@@ -15,18 +15,6 @@
   min-height: 390px;
 }
 
-.cm-gutters {
```

Comment:
I moved these to application.scss so they could be reused for Code Bridge. These are codemirror-specific selectors.